test/unit/orchestration/dhcp_test.rb

```diff
@@ -65,7 +65,8 @@ def teardown
   end
 
   test "provision interface DHCP records should contain filename/next-server attributes" do
-    ProxyAPI::TFTP.any_instance.expects(:bootServer).returns('192.168.1.1')
+    ProxyAPI::TFTP.any_instance.unstub(:bootServer)
```

Comment:
it is stubbed in `#setup`, and for some reason you can't override the original stubbed value, unless you unstub it first.
I am creating the same expectation in the next line. The problem is that it is called more than once, so I needed to change it anyway. I am not sure that `expects.at_least_once` is a necessary check in this test.